tasks/unapproved/UnapprovedRequestDescription.js

```diff
@@ -97,18 +97,27 @@ class UnapprovedRequestDescription {
   }
 
   __unresolvedAuthorsFor = () => {
+    const tagCommenters = this.__getConfigSetting("unapproved.tag.commenters", false)
+
     const { discussions } = this.request
 
+    const selectNotes = discussion => {
+      const [issue, ...comments] = discussion.notes
```

Comment:
`issuer`?

Reply:
We are in the notes context. Not referring to authors yet.